pkg/component/gardeneradmissioncontroller/poddisruptionsbudget.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gardeneradmissioncontroller
+
+import (
+	policyv1 "k8s.io/api/policy/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	gardenerutils "github.com/gardener/gardener/pkg/utils"
+)
+
+func (a admissioncontroller) podDisruptionBudget() *policyv1.PodDisruptionBudget {
+	if a.values.ReplicaCount <= 1 {
+		return nil
+	}
```

Comment:
This violates point 2 of https://github.com/gardener/gardener/blob/master/docs/development/component-checklist.md#high-availability--stability. I guess we can always deploy a PDB with `maxUnavailable=1` even with only `1` replica (I guess GOP will never deploy GAC with only `1`).